src/Compilers/Core/CodeAnalysisTest/FileSystem/PathUtilitiesTests.cs

```diff
@@ -296,29 +297,34 @@ public void IsSameDirectoryOrChildOfNegativeTests()
             Assert.Equal(false, PathUtilities.IsSameDirectoryOrChildOf(@"C:\A\B\C", @"C:\A\B\C\D"));
         }
 
-        [Theory]
-        [InlineData("test/data1.txt", true)]
-        [InlineData("test\\data1.txt", true)]
-        [InlineData("data1.txt", true)]
-        [InlineData("data1", true)]
-        [InlineData("data1\\", false)]
-        [InlineData("data1//", false)]
-        [InlineData(null, false)]
-        [InlineData("", false)]
-        [InlineData("  ", false)]
-        [InlineData("path/?.txt", false)]
-        [InlineData("path/*.txt", false)]
-        [InlineData("path/:.txt", false)]
-        [InlineData("path/\".txt", false)]
-        [InlineData(
-            "IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII" +
+        [Fact]
```

Comment:
Why did you have to move away from the theory here?

Reply:
Did you consider using `bool?` and letting `null` represent "platform dependent"?

Reply:
@agocke think you missed this comment.

Reply:
Woops, I did miss this comment. I'm not sure it's really worth it just to keep Theory around.